jetty-core/jetty-util/src/main/java/org/eclipse/jetty/util/FileID.java

```diff
@@ -65,9 +65,23 @@ public static String getFileName(URI uri)
     }
 
     /**
+     * <p>
```

Comment:
Strictly doesn't have anything to do with `MimeTypes`, but seems like a good cleanup.

Reply:
yeah, this is actually a pre-merge of #8924 (pending it going Green and being merged).  So will not be a part of this PR once merged.